src/Controllers/SaveApplicantController.php

```diff
@@ -31,40 +33,46 @@ public function __construct(ApplicantModel $applicantModel)
      * @param array $args The arguments array
      *
      * @return Response, will return the data from successfulRegister and the statusCode, via Json.
+     * @throws Exception
      */
-    public function __invoke(Request $request, Response $response, Array $args)
+    public function __invoke(Request $request, Response $response, array $args)
     {
         if ($_SESSION['loggedIn'] === true) {
             $data = ['success' => false, 'msg' => 'Application not saved', 'data' => []];
-            $statusCode = 401;
+            $statusCode = 500;
 
             $newApplicationData = $request->getParsedBody();
 
-            $applicant = $this->applicantModel->createNewApplicant(
-                $newApplicationData['name'],
-                $newApplicationData['email'],
-                $newApplicationData['phoneNumber'],
-                $newApplicationData['cohortId'],
-                $newApplicationData['whyDev'],
-                $newApplicationData['codeExperience'],
-                $newApplicationData['hearAboutId'],
-                $newApplicationData['eligible'],
-                $newApplicationData['eighteenPlus'],
-                $newApplicationData['finance'],
-                $newApplicationData['notes']
-            );
+            try {
+                $applicant = new ApplicantEntity(
+                    $newApplicationData['name'],
+                    $newApplicationData['email'],
+                    $newApplicationData['phoneNumber'],
+                    $newApplicationData['cohortId'],
+                    $newApplicationData['whyDev'],
+                    $newApplicationData['codeExperience'],
+                    $newApplicationData['hearAboutId'],
+                    $newApplicationData['eligible'],
+                    $newApplicationData['eighteenPlus'],
+                    $newApplicationData['finance'],
+                    $newApplicationData['notes']
+                );
+            } catch (Exception $e) {
+                return $response->withStatus(400);
```

Comment:
no message that explains why it failed? always send back an explanation. the exception you have caught will have a message in it that you can send back